VideoWeb/VideoWeb.AcceptanceTests/Pages/HearingRoomPage.cs

```diff
@@ -15,6 +15,7 @@ public static class HearingRoomPage
         public static readonly By ToggleHandRaised = By.Id("toggle-hand-raised-img");
         public static readonly By HandRaised = By.ClassName($"fa-hand-paper");
         public static readonly By ParticipantPanel = By.Id($"panelList");
-        public static By InterPreterName(string displayName) => By.XPath($"//span[contains(text(),'{displayName}')]");
+        //public static By InterPreterName(string displayName) => By.XPath($"//span[contains(text(),'{displayName}')]");
+        public static By InterPreterName(string displayName) => By.XPath($"//*[@id='panelList']/div[7]/span[contains(text(),'{displayName}')]");
```

Comment:
Can't you search only on displayname/

This appears as if it could change often, my workaround is to have two locators, first to the panel list and then a second to the displayname, so that if the div index changes then you can still find displayname.

Reply:
Could you do this change in another PR as this is not related to this particular story?